mk/ml.rc.in

```diff
@@ -0,0 +1,43 @@
+#include <Windows.h>
+
+1 VERSIONINFO
+    FILEVERSION    @ML_VERSION@,@ML_PATCH@

```

Comment:
There might be a problem here too. These versions need to be comma separated. But are we now adding a dot separated version?

Reply:
This one is ok.. ML_VERSION is defined in `cmake/functions.cmake` as
```
  if(${ML_VERSION_STR} MATCHES "([0-9.]+)")
    set(ML_VERSION ${CMAKE_MATCH_1})
  endif()
  string(REPLACE "." "," ML_VERSION "${ML_VERSION}")
```

Reply:
OK, great, sorry I missed that.